spec/services/update_veteran_attribs_service_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+describe UpdateVeteranAttribsService do
+  let(:veteranBob) { create(:veteran) }
+  let(:veteranSam) { create(:veteran) }
+  let(:appeal) { create(:appeal, veteran_file_number: veteranBob.file_number) }
+  let(:appeal2) { create(:appeal, veteran_file_number: veteranSam.file_number) }
```

Comment:
Easier and shorter to read
```suggestion
  let(:appeal) { create(:appeal, veteran: veteranBob) }
  let(:appeal2) { create(:appeal, veteran: veteranSam) }
```